IS1001Test.py

```diff
@@ -12,12 +12,356 @@
+                'description': 'The lifetime in seconds of the access token'
+              },
+              'access_token': {
+                'type': 'string',
+                'description': 'Access Token to be used in accessing protected endpoints'
+              },
+              'token_type': {
+                'type': 'string',
+                'enum': ['Bearer'],
+                'description': 'The type of the Token issued'
+              },
+              'refresh_token': {
+                'type': 'string',
+                'description': 'Refresh Token to be used to obtain further Access Tokens'
+              },
+            },
+            'required': ['access_token', 'expires_in', 'token_type'],
+            'type': 'object',
+            'description': 'OAuth2 Response for the request of a Bearer Token',
+          },
+          400: {
+            'title': 'Token Error Response',
+            '$schema': 'http://json-schema.org/draft-07/schema#',
+            'properties': {
+              'error': {
+                'type': 'string',
+                'description': 'Error Type'
+              },
+              'error_uri': {
+                'type': 'string',
+                'description': 'A URI identifying a human-readable web page with information about the error'
+              },
+              'error_description': {
+                'type': 'string',
+                'description': 'Human-readable ASCII text providing additional information'
+              }
+            },
+            'required': ['error'],
+            'type': 'object',
+            'minItems': 1,
+            'description': 'Object defining error type and description',
+          }
+        },
+      }],
+    }
+  }
+
+  Example "apis" Object:
+  {
+      'auth': {
+          'raml': 'AuthorizationAPI.raml',
+          'version': 'v1.0',
+          'hostname': 'example.co.uk',
+          'port': 80,
+          'spec_branch': 'v1.0-dev',
+          'spec_path': 'cache/is-10',
+          'base_url': 'http://example.co.uk:80',
+          'name': 'Authorization API',
+          'ip': '127.0.1.1',
+          'url': 'http://example.co.uk:80/x-nmos/auth/v1.0/',
+          'spec': <Specification.Specification object at 0x7f5b3b0719b0>
+      }
+  }
+  """
+
     def __init__(self, apis):
         GenericTest.__init__(self, apis)
+
+        self.url = self.apis[AUTH_API_KEY]["url"]
+        self.bearer_token = None
+        self.client_data = None
+
+    def set_up_tests(self):
+        try:
+            signup_data = {
+                'username': AUTH_USERNAME,
+                'password': AUTH_PASSWORD,
+                'is04': 'read',
+                'is05': 'write'
+            }
+            status, response = self.do_request(
+                method="POST", url=self.url + 'signup', data=signup_data
+            )
+
+            if status is not True:
+                raise Exception
+
+        except Exception:
+            print("""

```

Comment:
Can this raise an exception rather than print so it shows the issue in the GUI? NMOSInitException may be appropriate.